### ISSUE TITLE
Example code for data generation

### ISSUE THREAD

**Issue by ashar6194** [RichardYang40148/MidiNet#6]
Hi Richard, 

Thanks for providing the code. Can you also provide a working example where you generate sound from **scratch** using a learned model?
I am new to audio domain, and it would be a great help.

I am trying to use the function generation_test() in the _utils.py_ file to generate data from scratch. But it is throwing an error:
FailedPreconditionError: Attempting to use uninitialized value g_h0_prev_conv/w

**Comment by RichardYang40148** [comment]
Hi! Can you load the trained model?
P.S. Please make sure you got the compatible Tensorflow version with the released code (0.12)

**Comment by RichardYang40148** [comment]
Hi Ashar, 
I guess this might solve your issue:
Try to change the dataset name from "default into "MidiNet_v1" in line18 of model.py
"https://github.com/RichardYang40148/MidiNet/blob/acfc779610a2ba926f4dcb6ce6ac2200c46a1169/v1/model.py#L18

**Comment by jliuliu** [comment]
hi, I'm very interest to your model, and can you give input data example, <data_X, prev_X, data_y> format, thank you!

**Comment by ashar6194** [comment]
Hi Richard,
 
I changed my Tensorflow version to 0.12. And yes, I downloaded the trained model you have provided in your drive. But I am still facing the error. Is there a mini-support document or a 'readme' to run only the generation pipeline using a learned model of MidiNEt?

Is there a change in the code/architecture versions between the trained model and the latest code??
I am getting the following error on the step:

**InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [77] rhs shape= [64]**

_Line of Code-_
**self.saver.restore(self.sess, os.path.join(checkpoint_dir, ckpt_name))**
This is line 362 of the file model.py
